src/gsi/setuprad.f90

```diff
@@ -1047,7 +1052,7 @@ subroutine setuprad(obsLL,odiagLL,lunin,mype,aivals,stats,nchanl,nreal,nobs,&
            endif
         endif
         ! Screening for cold-air outbreak area (only applied to MW for now)
```

Comment:
Does this mean the cao check is only performed when precip is turned on?

Reply:
@ADCollard  Yes, the detection of CAO requires to use all of the five hydrometeors.